modules/core/src/main/java/org/apache/ignite/internal/managers/discovery/DiscoveryMessageFactory.java

```diff
@@ -48,7 +52,8 @@
 public class DiscoveryMessageFactory implements MessageFactoryProvider {
     /** {@inheritDoc} */
     @Override public void registerAll(MessageFactory factory) {
-        factory.register((short)-100, InetAddressMessage::new, new InetAddressMessageSerializer());
+        factory.register((short)-101, InetAddressMessage::new, new InetAddressMessageSerializer());
```

Comment:
`InetSocketAddressMessage` extends `InetAddressMessage`. Its number should be higher. We use negative numbering here, so -101 id bigger that -100 as order. I sugget setting direct type -100 for `InetAddressMessage` and -101 for `InetSocketAddressMessage`